src/Bindings/DiffAPIDesc.lua

```diff
@@ -129,14 +129,17 @@ end
 -- a_FunctionDoc is a single documentation item for a function, as loaded from ToLua++'s parser
 local function functionDescMatchesDocs(a_FunctionDesc, a_FunctionDoc)
 	-- Check the number of parameters:
-	local numParams
+	local numParams = 0
 	local numOptionalParams = 0
 	if (not(a_FunctionDesc.Params) or (a_FunctionDesc.Params == "")) then
 		numParams = 0
 	else
-		_, numParams = string.gsub(a_FunctionDesc.Params, ",", "")
-		numParams = numParams + 1
-		_, numOptionalParams = string.gsub(a_FunctionDesc.Params, "%b[]", "")
+		for a,b in pairs(a_FunctionDesc.Params) do
```

Comment:
The usual Lua convention is to use `k` and `v` for `pairs`-based iteration ("key" and "value"). Also, missing a space after the comma.